### PR TITLE
[#363] Move Model3DViewer to SDK ver2.0

### PR DESCRIPTION
**Pull request by mrtraser**

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-364.gearboxjs.preview.cogniteapp.com

File: src/components/Model3DViewer/Model3DViewer.tsx [hunk 1]
```diff
@@ -248,18 +259,23 @@ export class Model3DViewer extends React.Component<Model3DViewerProps> {
...
     }
 
-    const { items } = await sdk.ThreeD.listAssetMappings(modelId, revisionId, {
+    return await this.context!.assetMappings3D.list(modelId, revisionId, {
       assetId,
-    });
+    }).autoPagingToArray();
+  }
+
+  private async getRevision() {
```
Comment (xitman78) [inline src/components/Model3DViewer/Model3DViewer.tsx]: no need in async here since the function returns a promise

File: src/mocks/threeD.ts [hunk 2]
```diff
@@ -61,11 +63,35 @@ export function createFakeViewer({
...
   };
 
   return cache[project];
 }
+
+export const fakeModel3DViewerClient: API = {
+  // @ts-ignore
+  revisions3D: {
+    // @ts-ignore
+    retrieve: (): Revision3D => revision3D,
```
Comment (xitman78) [inline src/mocks/threeD.ts]: shouldn't the function be async or return a promise?

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-364.gearboxjs.preview.cogniteapp.com

File: src/mocks/threeD.ts (hunk 2 above)
Comment (mrtraser) [reply src/mocks/threeD.ts]: Fixed

File: src/components/Model3DViewer/Model3DViewer.tsx (hunk 1 above)
Comment (mrtraser) [reply src/components/Model3DViewer/Model3DViewer.tsx]: true, fixed

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-364.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-364.gearboxjs.preview.cogniteapp.com

**Comment by mrtraser** [comment]
Closes #363

**Pull request merged**